### DISCUSSION TITLE
How to backup?

### DISCUSSION THREAD

**Discussion by Bogey** [Budibase/budibase#1912] [Help] [answered]
I've been wondering what would be the best way to backup Budibase.
A backup and restore for the app can easily be created via the import/export mechanism.
But what about the data that's been entered in apps?

In the CouchDB documentation I read that: "The simplest and easiest approach for CouchDB backup is to use replication. [..] However, you can also copy the actual .couch files from the CouchDB data directory (by default, data/) at any time, without problem."

I'm running budibase in Docker with persistent volumes for CouchDB data and Minio data on a local NAS. Will a simple daily backup of these volumes do?

**Comment by shogunpurple** [comment, marked as answer]
@Bogey that's correct - the `/opt/couchdb/data` directory backup on your NAS should have you covered. If a total failure occurred, you could remount that drive into couch and it would work as expected.

**Reply by Bogey** [reply]
Thanks for your confirmation.
Another question also regarding backup/storage: what for/how is Minio used?

**Reply by mike12345567** [reply]
MinIO is currently used as the file storage system for your Budibase installation, for example the `budibase-client` and the various modules required for each app are stored there (we store these per app so that as you upgrade your Budibase version apps built with older versions aren't broken) as well as attachments, images etc.

**Reply by Bogey** [reply]
Ahaa I see, so putting that storage into the daily backup routine should also cover that part.
